render/detailed/connections.go

```diff
@@ -145,7 +145,7 @@ func outgoingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 				if !ok {
 					continue
 				}
-				key := newConnection(n, node, port, remoteEndpointID, localAddr)
```

Comment:
The "The Internet (Inbound connections)" node in the UI.

Recall that outbound connections of a node are connections from that node to some other node. We list these in the adjacency.

From the perspective of the Internet node, the outbound connections / adjacency contains connections to containers etc. From the perspective of the user, these are inbound connections from the Internet into the system.